.travis.yml

```diff
@@ -1,17 +1,36 @@
 language: ruby
-os: osx
-osx_image: xcode8.3
-rvm: system
+matrix:
+  fast_finish: true
+  include:
+    - os: osx
+      osx_image: xcode8.3
+      rvm: system
+    - os: linux
+      sudo: required
+      dist: trusty
+      rvm: 2.3.3
 
 before_install:
   - export HOMEBREW_DEVELOPER=1
+  - export HOMEBREW_NO_AUTO_UPDATE=1
   - if [ -f ".git/shallow" ]; then
       travis_retry git fetch --unshallow;
     fi
-  - HOMEBREW_REPOSITORY="$(brew --repo)"
-  - sudo chown -R "$USER" "$HOMEBREW_REPOSITORY"
-  - git -C "$HOMEBREW_REPOSITORY" fetch;
-  - git -C "$HOMEBREW_REPOSITORY" reset --hard origin/master;
+  - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then
+      HOMEBREW_REPOSITORY="$(brew --repo)";
+      sudo chown -R "$USER" "$HOMEBREW_REPOSITORY";
+      git -C "$HOMEBREW_REPOSITORY" fetch;
+      git -C "$HOMEBREW_REPOSITORY" reset --hard origin/master;
+    else
+      HOME=/home/linuxbrew;
```

Comment:
@sjackman Is there a reason there's `/home/linuxbrew` and `/home/.linuxbrew`?

Reply:
`HOME=/home/linuxbrew` and `HOMEBREW_PREFIX=/home/.linuxbrew/.linuxbrew`

Reply:
@sjackman can you check you've not typoed that?

Reply:
Yep, that's a typo. Should be
`HOME=/home/linuxbrew` and
`HOMEBREW_PREFIX=/home/linuxbrew/.linuxbrew`

Reply:
Ok, thanks.